Scripts/LineageOS-14.1/Functions.sh

```diff
@@ -34,7 +34,7 @@ buildDevice() {
 	cd "$DOS_BUILD_BASE";
 	export OTA_KEY_OVERRIDE_DIR="$DOS_SIGNING_KEYS/$1";
 	pkill java && sleep 10; #XXX: ugly hack
-	breakfast "lineage_$1-user" && mka target-files-package otatools && processRelease $1 true $2;
+	breakfast "lineage_$1-user" && mka -j${DOS_MAX_THREADS_BUILD} target-files-package otatools && processRelease $1 true $2;
```

Comment:
mka already defaults to all cores

Reply:
Yes and this change allows it to decrease it. It's not needed when you want to use all cores but this not always wanted - at least for me